src/Misc/layoutbin/en-US/strings.json

```diff
@@ -200,22 +191,41 @@
-    "bin/Agent.Listener configure --unattended --url https://myaccount.visualstudio.com --agent myagent --pool default --nostart --acceptteeeula --auth PAT --token o4u5... --work D:\\agent_work",
-    "bin/Agent.Listener run",
+    "Unattend configuration for VSTS with PAT authentication",
+    "Win  : config.cmd --unattended --url https://myaccount.visualstudio.com --auth PAT --token o4u5... --pool default --agent myagent",
+    "Unix : ./config.sh --unattended --acceptteeeula --url https://myaccount.visualstudio.com --auth PAT --token o4u5... --pool default --agent myagent",
     "",
     "Unattend configuration for on-premise TFS with integrated (windows) and (Linux/OSX) negotiate authentication",
-    "bin/Agent.Listener configure --unattended --url http://mylocaltfsserver:8080/tfs --agent myagent --pool default --nostart --acceptteeeula --auth integrated",
-    "bin/Agent.Listener configure --unattended --url http://mylocaltfsserver:8080/tfs --agent myagent --pool default --nostart --acceptteeeula --auth negotiate --username DEV-BOX\\developer --password MyPassword"
+    "Win  : config.cmd --unattended --url http://mylocaltfsserver:8080/tfs --auth Integrated --pool default --agent myagent",
+    "Unix : ./config.sh --unattended --acceptteeeula --url http://mylocaltfsserver:8080/tfs --auth Negotiate --username DEV-BOX\\developer --password MyPassword --pool default --agent myagent",
+    "",
+    "Unattend configuration for VSTS with PAT authentication and replace existing agent with same agent name",
+    "Win  : config.cmd --unattended --url https://myaccount.visualstudio.com --auth PAT --token o4u5... --pool default --agent myagent --replace",
+    "Unix : ./config.sh --unattended --acceptteeeula --url https://myaccount.visualstudio.com --auth PAT --token o4u5... --pool default --agent myagent --replace",
+    "",
+    "Unattend configuration for VSTS with PAT authentication and specify agent work directory",
+    "Win  : config.cmd --unattended --url https://myaccount.visualstudio.com --auth PAT --token o4u5... --pool default --agent myagent --work D:\\agent_work",
+    "Unix : ./config.sh --unattended --acceptteeeula --url https://myaccount.visualstudio.com --auth PAT --token o4u5... --pool default --agent myagent --work usr/local/agent_work",
+    "",
+    "Unattend configuration for on-premise TFS with integrated authentication and configure agent as Windows service run as NetworkService",
+    "Win  : config.cmd --unattended --url http://mylocaltfsserver:8080/tfs --auth Integrated --pool default --agent myagent --runasservice",
+    "",
+    "Unattend configuration for on-premise TFS with integrated authentication and configure agent as Windows service run as a domain account",
+    "Win  : config.cmd --unattended --url http://mylocaltfsserver:8080/tfs --auth Integrated --pool default --agent myagent --runasservice --windowslogonaccount DEV-BOX\\developer --windowslogonpassword MyPassword",
```

Comment:
Existing issue, but i might be good to cleanup now. I think `DEV-BOX\developer` will look unprofessional to IT pros. `DOMAIN\USER_NAME` seems better to me.